tests/testapp/tests/test_client.py

```diff
@@ -32,8 +32,8 @@ class TestForm(forms.Form):
 
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
-    url(r'^view/', view),
+    re_path(r'^admin/', admin.site.urls),
+    re_path(r'^view/', view),
```

Comment:
what about replacing `re_path` with the `path` function?
there is nothing special that requires regular expression parsing

Reply:
I was too lazy to get rid of regular expressions :)